src/app/auth/google/google.component.ts

```diff
@@ -227,6 +227,7 @@ export const getStateParamsFromGoogle = (
     deleteKey: false,
     derivedPublicKey: '',
     expirationDays: 0,
+    redirect_uri: '',
```

Comment:
we're also going to need to update `google-drive.service.ts` around line 97 to add `redirectUri: this.globalVars.redirectUri` so it gets fed into the google flow and we still have the param when we get back from the google flow.

Reply:
damn good catch, yeah i tried the google flow with popup but did not do the full page redirect + google flow. Just test and confirmed it was broken and this fixes it.